### PR TITLE
use the new payoutMethodStatus property to correctly flag payout status in payment tab

### PR DESCRIPTION
**Pull request by bystam**
Start using the new `payoutMethodStatus` to both show it in the UI and also show/hide the payout controls of the UI based on it rather than an `if Sweden`-hack.

[x] `src/changelog.ts` updated

### CODE REVIEW

**Review by AlexSJ1990 — DISMISSED** [review]
✅

**Review by AlexSJ1990 — DISMISSED** [review]
Nice!

File: src/changelog.ts [hunk 1]
```diff
@@ -12,17 +12,22 @@ export interface Change {
...
   },
   {
-    date: '2020-01-18',
+    date: '2021-01-18',
     change: 'Add ability to generate direct debit link for all members',
     authorGithubHandle: 'vonElfvin',
   },
   {
-    date: '2020-01-18',
+    date: '2021-01-18',
```
Comment (FredrikAreschoug) [inline src/changelog.ts:30]: :laughing:

**Review by AlexSJ1990 — APPROVED** [review]
✅

**Pull request merged**